x-pack/plugins/index_management/common/types/data_streams.ts

```diff
@@ -5,20 +5,20 @@
  * 2.0.
  */
 
+import {
+  ByteSize,
+  IndicesDataLifecycleWithRollover,
+  IndicesDataStream,
+  IndicesDataStreamsStatsDataStreamsStatsItem,
+  Metadata,
+} from '@elastic/elasticsearch/lib/api/types';
+
 interface TimestampFieldFromEs {
   name: string;
 }
 
 type TimestampField = TimestampFieldFromEs;
 
-interface MetaFromEs {
```

Comment:
I refactored some of the server-side code to leverage the types from `@elastic/elasticsearch` rather than our own custom interfaces